File: src/BukkitPE/level/particle/ItemBreakParticle.php
<?php
namespace BukkitPE\level\particle;

use BukkitPE\math\Vector3;
use BukkitPE\item\Item;

class ItemBreakParticle extends GenericParticle{
	public function __construct(Vector3 $pos, Item $item){
		parent::__construct($pos, Particle::TYPE_ITEM_BREAK, ($item->getId() << 16) | $item->getDamage());
	}
}

File: src/main/java/net/BukkitPE/level/particle/ItemBreakParticle.java
package net.BukkitPE.level.particle;

import net.BukkitPE.item.Item;
import net.BukkitPE.math.Vector3;

/**
 * Created on 2015/11/21 by xtypr.
 * Package net.BukkitPE.level.particle in project BukkitPE .
 */
public class ItemBreakParticle extends GenericParticle {
    public ItemBreakParticle(Vector3 pos, Item item) {
        super(pos, Particle.TYPE_ITEM_BREAK, (item.getId() << 16) | item.getDamage());
    }
}
